contracts/shiden34/lib.rs

```diff
@@ -130,6 +137,7 @@ pub mod shiden34 {
             },
             prelude::string::String as PreludeString,
```

Comment:
Remove PreludeString alias on all places. OB is now using ink::prelude::String as well